src/views/widget/index.js

```diff
@@ -0,0 +1,23 @@
+import { h } from "preact";
+import { useState } from "preact/hooks";
+// import { WidgetWrapper, Title } from "./styled";
+
+import Modal from "../../components/modal";
+import Button from "../../components/button";
+
+const Widget = () => {
+  const [isModalOpen, setModalIsOpen] = useState(false);
+
+  const toggleModalVisibility = () => {
```

Comment:
How do we make this available from the outside? 

I want users to be able to have their custom html to open the modal. 

e.g. I have a link that opens the widget

```html
<a href="#" onclick="Widget.open()">open</a>
```

Reply:
I was going with the initial flow of adding the button to the HTML via the classname. So, I am thinking of completing that first.
```HTML
...
<div data-widget-id=lighitning-payments />
...
```

I will check on how to expose the `Widege.open()` from the HTML parts.